### PR TITLE
[ISSUE #3789] optimize: Tag the name of  consuming thread whith consumeGroup.

### PR DESCRIPTION
**Pull request by XiaoyiPeng**
**Make sure set the target branch to `develop`**

## What is the purpose of the change

For issue #3789

## Verifying this change

- **Unit tests**
- **Pictures below.**

image

image

### CODE REVIEW

**Review by dongeforever — APPROVED** [review]
LGTM

**Review by ni-ze — APPROVED** [review]

**Review by duhenglucky — APPROVED** [review]

File: client/src/main/java/org/apache/rocketmq/client/impl/consumer/ConsumeMessageConcurrentlyService.java [hunk 1]
```diff
@@ -70,13 +70,19 @@ public ConsumeMessageConcurrentlyService(DefaultMQPushConsumerImpl defaultMQPush
         this.consumerGroup = this.defaultMQPushConsumer.getConsumerGroup();
         this.consumeRequestQueue = new LinkedBlockingQueue<Runnable>();
 
+        String consumeThreadPrefix = null;
+        if (consumerGroup.length() > 100) {
```
Comment (caigy) [inline client/src/main/java/org/apache/rocketmq/client/impl/consumer/ConsumeMessageConcurrentlyService.java:74]: Is it `100` too long for displaying?

**Review by caigy — APPROVED** [review]

File: client/src/main/java/org/apache/rocketmq/client/impl/consumer/ConsumeMessageConcurrentlyService.java (hunk 1 above)
Comment (XiaoyiPeng) [reply client/src/main/java/org/apache/rocketmq/client/impl/consumer/ConsumeMessageConcurrentlyService.java:74]: Thanks for your review.

1. The maximum length of a `topic` name is **127**. The business-side naming `consumeGroup` usually contains some or all of the information for the `topic name`, so this length value 100 maybe relatively reasonable.
2. In addition, although the maximum length of a `topic` name is **127**, in the actual business, users may not take such a long name. Similarly, the naming of consumer groups is also the same.
3. You can type 100 characters into your text editor and it doesn't look so bad.

Comment (XiaoyiPeng) [reply client/src/main/java/org/apache/rocketmq/client/impl/consumer/ConsumeMessageConcurrentlyService.java:74]: > Is it `100` too long for displaying?

Thanks for your review.

The maximum length of a topic name is 127. The business-side naming consumeGroup usually contains some or all of the information for the topic name, so this length value 100 maybe relatively reasonable.
In addition, although the maximum length of a topic name is 127, in the actual business, users may not take such a long name. Similarly, the naming of consumer groups is also the same.
You can type 100 characters into your text editor and it doesn't look so bad.

**Review by panzhi33 — APPROVED** [review]

**Pull request merged**